catalogs/catalog-lakehouse-iceberg/src/main/java/com/datastrato/gravitino/catalog/lakehouse/iceberg/converter/ToIcebergTypeVisitor.java

```diff
@@ -52,6 +54,19 @@ public static <T> T visit(Type type, ToIcebergTypeVisitor<T> visitor) {
     } else if (type instanceof Types.ListType) {
       Types.ListType list = (Types.ListType) type;
       return visitor.array(list, visit(list.elementType(), visitor));
+    } else if (type instanceof Types.StructType) {
+      Types.StructType struct = (Types.StructType) type;
+      Types.StructType.Field[] fields = struct.fields();
+      List<IcebergColumn> columns =
+          Arrays.stream(fields)
+              .map(
+                  field -> {
+                    return ConvertUtil.fromGravitinoField(field, 0);
```

Comment:
The id is always zero?

Reply:
The id will not be used in the subsequent logic, visitor.struct() will generate this ID according to iceberg's generation rules. 
My idea is that since the generated id here will not be used later, assigning a value of 0 directly to the id will make the code more concise
<img width="901" alt="image" src="https://github.com/datastrato/gravitino/assets/58762426/fe9bfe46-281f-4674-83fd-48c346b06d9c">

Reply:
Do you think it is necessary to add generated ID logic in this method, though these ID will never be used?

Reply:
The 0 of id passed here will never be used in the subsequent logic, so It's not necessary to generate id here.
As you mentioned, it will be much appreciate if you can raise an issue to discuss the refactor of previous code and abandon the use of ID.